### DISCUSSION TITLE
v2.4.2-Beta

### DISCUSSION THREAD

**Discussion by Gkiokan** [Gkiokan/ps4-remote-pkg-sender#7] [Show and tell]
Further Improvements and optimizing Request Handling.  
  
Added custom timeout and interval values for updating task progress  
Added deeper logging on PS4 Operations and Responses  
Added Check Server Button on the Progress Center
Added Check PS4 Connection on the Progress Center  
Added dedicated [Changelog](https://github.com/Gkiokan/ps4-remote-pkg-sender/blob/master/Changelog.md)
Added dedicated [Troubleshooting Guide](https://github.com/Gkiokan/ps4-remote-pkg-sender/blob/master/Troubleshoot.md)
Added Changelog and Troubleshooting guide into the Application Menu  
Added extendable Table row on Server files to see file path and url  

<hr /><em>This discussion was created from the release <a href='https://github.com/Gkiokan/ps4-remote-pkg-sender/releases/tag/v2.4.2'>v2.4.2-Beta</a>.</em>

**Comment by HardRom** [comment]
Portable 2.4.2 won't start server, check heartbeat fails
![image](https://user-images.githubusercontent.com/277477/148125536-9335543f-9542-4efc-842c-089b9b56965f.png)
Extracted 2.4.2 doesn't load the gui after the uncaught reference error

Always as admin, restarted RPI with each attempt.

**Reply by Gkiokan** [reply]
You still get the uncaught reference error?

Can you check your server ip and port (maybe a different one) and klick on refresh server or the i/o button to reload it.

With RPI I mean the Remote Package Installer on your PS4.

**Reply by Gkiokan** [reply]
After that try to check the connection against Server and PS4 on the Processing Center.

![image](https://user-images.githubusercontent.com/7249224/148126620-555ed4f1-6953-4905-a6e3-67d3e839cf62.png)
![image](https://user-images.githubusercontent.com/7249224/148126749-94e9d670-1abd-4717-bb58-592f8e22b127.png)

**Reply by HardRom** [reply]
"With RPI.." yes, I understood that.

Uncaught reference everytime.  I checked firewall settings and they allow private and public access.  I also tried changing the port from 8337 default to 8080, 8081, 4321 with the same result using the portable.

I can give you remote access if you'd like to see yourself- PM me

**Reply by HardRom** [reply]
Server is not running, each attempt to check it (portable) gives uncaught reference.  Heartbeat loads a new blank window.

**Reply by HardRom** [reply]
![image](https://user-images.githubusercontent.com/277477/148127044-517622d6-9dea-4dcd-a944-aaa1c5556148.png)

**Reply by Gkiokan** [reply]
Is this the unpacked version or the portable? If portable try the unpacked version.  Maybe I should just kill the portable version because it just makes more trouble then working.

Where can I PM you? Gimme PM on Discord if you are on it: Gkiokan#0735

**Comment by luongthien93** [comment]
Very nice and I think this is work better than the first version. Nice work. I'm looking the next ver from you.
